template/README.md

```diff
@@ -1,4 +1,4 @@
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [Redux](https://redux.js.org/) and [Redux Toolkit](https://redux-toolkit.js.org/) template.
+This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app), using the [CARTO](https://carto.com/developers/create-react-app) template.
```

Comment:
https://carto.com/developers/create-react-app is not available yet. Let's include when it's available

Reply:
Link removed